styles/main-styles.css

```diff
@@ -0,0 +1,245 @@
+@import url(./fonts.css);
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.background-image {
+  background: url("../images/back/castelmezzano-1979546 1.svg") no-repeat;
+  background-size: 100%;
+  min-height: 100%;
+  height: 900px;
+  margin: 0 auto;
+  padding: 51px 142px 0 142px;
```

Comment:
So, you need to align to the center your content  on different displays, in this case your's paddings doesn't work. Howevwe, you can add max-width:1220 px, according design and align to th center using margin:0 auto

Reply:
Done, but if I add "max-width: 1220px", in mobile view background-image shrinks in a half, so I added "with: 1440px", like in design. The same goes for footer.